querydsl-codegen/src/test/java/com/querydsl/codegen/EntitySerializerTest.java

```diff
@@ -78,7 +78,7 @@ public void No_Package() throws IOException {
     }
 
     @Test
-    public void OriginalCategory() throws IOException {
+    public void originalCategory() throws IOException {
```

Comment:
Cool, yes I didn't get what you meant with `camelCase_variant_in_lower_case_snake_case`.
I'm all for it.